code/workspaces/web-app/src/components/settings/UserPermissionsTableActionCells.spec.js

```diff
@@ -2,37 +2,40 @@ import React from 'react';
 import { createShallow } from '@material-ui/core/test-utils';
```

Comment:
Do you need to remove this import?

Reply:
Not yet - there's still a couple of tests in that file I haven't managed to migrate yet because of interactions with clicking on things.

Reply:
Ah gotcha, that's fine